subprojects/docs/src/docs/userguide/migrating_from_maven.adoc

```diff
@@ -0,0 +1,518 @@
+
+NOTE: You can use this feature to apply the `<dependencyManagement>` information from any dependency's POM to the Gradle build, even those that don't have a packaging type of `pom`. However, `platform()` and `enforcedPlatform()` will ignore any dependencies declared in the `<dependencies>` block.
+
+== Migrating multi-module builds (project aggregation)
+
+Maven's multi-module builds map nicely to Gradle's <<multi_project_build#multi_project_build,multi-project builds>>.
+Try the corresponding link:{guidesUrl}/creating-multi-project-builds/[tutorial] to see how a basic multi-project build is set up.
+
+To migrate a multi-module Maven build, simply follow these steps:
+
+ 1. Create a settings script that matches the `<modules>` block of the root POM.
++
+For example, this `<modules>` block:
++
+[source,xml]
+----
+<modules>
+    <module>simple-weather</module>
+    <module>simple-webapp</module>
+</modules>
+----
++
+can be migrated by adding the following line to the settings script:
++
+[source,groovy]
+.settings.gradle
+----
+include 'simple-weather', 'simple-webapp'
+----
+ 2. Replace cross-module dependencies with <<dependency_types#sub:project_dependencies,project dependencies>>.
+ 3. Replicate project inheritance with <<multi_project_builds#sec:cross_project_configuration,cross-project configuration>>.
++
+This basically involves creating a root project build script that injects shared configuration into the appropriate subprojects.
+
+One notable feature that's missing is a standard way to share dependency versions across projects in a multi-project build.
+A common approach is to use <<writing_build_scripts#sec:extra_properties,extra properties>> in the root build script to store the versions, since those properties are visible from subprojects as well.
+
+== Migrating Maven profiles and properties
+
+Maven allows you parameterize builds using properties of various sorts.
+Some are read-only properties of the project model, others are user-defined in the POM.
+It even allows you to treat system properties as project properties.
+
+Gradle has a similar system of project properties, although it differentiates between those and system properties.
+You can, for example, define properties in:
+
+* the build file
+* a `gradle.properties` file in the root project directory
+* a `gradle.properties` file in the `$HOME/.gradle` directory
+
+Those aren't the only options, so if you are interested in finding out more about how and where you can define properties, check out the <<build_environment#build_environment,Build Environment>> chapter.
+
+One important piece of behavior you need to be aware of is what happens when the same property is defined in both the build file and one of the external properties files: the build file value takes precedence.
+Always.
```

Comment:
command-line properties take precedence here over those in the build file, so maybe this fact is notable here?

Reply:
I don't think that's true. Are you sure? My testing suggests otherwise.

Reply:
Huh... so is this preamble here wrong? https://docs.gradle.org/current/userguide/build_environment.html#header

Reply:
@eriwen I don't know. I would assume it's correct. However, it doesn't mention extra properties defined in the build script, so doesn't conflict with the information here.